libgalois/src/PropertyGraph.cpp

```diff
@@ -148,17 +148,10 @@ struct PropertyColumn {
 };
 
 /// Assumes all boolean or uint8 properties are types
-katana::Result<katana::NUMAArray<katana::PropertyGraph::EntityTypeID>>
+katana::Result<katana::NUMAArray<katana::EntityTypeID>>
```

Comment:
Why not move this function as a generic API along with the EntityTypeManager. It's another way to create entity types.